cluster/gce/util.sh

```diff
@@ -1550,6 +1550,8 @@ CNI_DIR: $(yaml-quote ${WINDOWS_CNI_DIR})
 CNI_CONFIG_DIR: $(yaml-quote ${WINDOWS_CNI_CONFIG_DIR})
 WINDOWS_CNI_STORAGE_PATH: $(yaml-quote ${WINDOWS_CNI_STORAGE_PATH})
 WINDOWS_CNI_VERSION: $(yaml-quote ${WINDOWS_CNI_VERSION})
+WINDOWS_CONTAINER_RUNTIME: $(yaml-quote ${WINDOWS_CONTAINER_RUNTIME:-})
+WINDOWS_CONTAINER_RUNTIME_ENDPOINT: $(yaml-quote ${WINDOWS_CONTAINER_RUNTIME_ENDPOINT:-})
```

Comment:
Other entries in this list omit the `:-`. I don't expect it makes a difference, but for consistency please remove `:-` from these entries also.